build-requirements.txt

```diff
@@ -0,0 +1,4 @@
+bandit
+flake8
+mypy
+lint
```

Comment:
Is it intentional to have replaced pylint with lint?

Reply:
no that is a typo. I'll fix it